packages/stable/src/api/spatial/types.ts

```diff
@@ -0,0 +1,141 @@
+import { CogniteExternalId, CogniteInternalId, Limit } from '@cognite/sdk-core';
+
+export type GeometryType = 'wkt' | 'geojson';
+
+export type AttributeType =
```

Comment:
I think you have to export these to the main types.ts file so others can access them in a consistent manner.

Reply:
exported after importing from this file from src/types.ts as discussed on slack thread.
@andersfylling